.vitepress/components/Translator.ts

```diff
@@ -0,0 +1,34 @@
+import {useData} from "vitepress";
+import {messages} from "../theme/enhancements/i18n/i18n";
+
+export function useTranslator() {
+
+    const FALLBACK_LANG = 'en';
+
+    const { lang } = useData();
+
+    function t(translationKey: String): String {
+        let langMap;
+
+        if(Object.keys(messages).indexOf(lang.value) != -1) {
+            langMap = messages[lang.value];
+        } else {
+            langMap = messages[FALLBACK_LANG];
+        }
+
+        let result = langMap;
+
+        translationKey.split(".").forEach(key => {
```

Comment:
can u make a constant for `.` to make the code more readable?

Reply:
I have 👏